build.gradle

```diff
@@ -28,6 +28,9 @@ repositories {
     maven {
         url 'http://dogforce-games.com/maven'
     }
+    maven {
+        url 'http://minecraft.curseforge.com/api/maven/'
```

Comment:
No curseforge maven is not properly implemented. Untill they improve their implementation please find their official maven.